pallets/phala/src/fat.rs

```diff
@@ -206,24 +201,34 @@ pub mod pallet {
 		}
 
 		#[pallet::weight(0)]
-		pub fn upload_code_to_cluster(
+		pub fn cluster_upload_resource(
 			origin: OriginFor<T>,
-			code: Vec<u8>,
 			cluster_id: ContractClusterId,
+			resource_type: ResourceType,
+			resource_data: Vec<u8>,
```

Comment:
Consider to use `BoundedVec`?

Reply:
I don't think it's necessary since the bound is dependent.